stack.yaml

```diff
@@ -13,3 +13,7 @@ packages:
 
 nix:
   shell-file: shell.nix
+
+#extra-deps:
```

Comment:
What are these for?

Reply:
If you try to compile with updated (but commented out) `nightly-2019-09-28`, you'll need to add these.